### PR TITLE
increased the cloudwatch threshold of ApiGateway4XXErrorAlarm and ApiGateway5XXErrorAlarm

### PR DESCRIPTION
**Pull request by nirojshrestha019**
Issue #, if available:

Description of changes:

Checklist:

* [ ] Have you successfully deployed to an AWS account with your changes?
* [ ] Have you written new tests for your core changes, as applicable?

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by nirojshrestha019** [comment]
I have only made changes to cloudformation/alarms.yaml file. All the other changes are fetched from the upstream awslabs repo.

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,33 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [2.3.0-smart](https://github.com/awslabs/fhir-works-on-aws-deployment/compare/v2.2.0-smart...v2.3.0-smart) (2021-12-13)
```
Comment (lquanx) [inline CHANGELOG.md]: Isn't it better to have a separate PR for 2.3.0 ? It would be easier to review the PR.

Comment (nirojshrestha019) [reply CHANGELOG.md]: I have only made changes to cloudformation/alarms.yaml file in this PR. I am okay with having a separate PR to pull the upstream if everyone is okay with that.

Comment (lwarren01) [reply CHANGELOG.md]: I agree. We probably want to handle the upgrade separately from this task.

**Comment by lquanx** [comment]
If we merge with  v2.3.0. All other 6 submodules have to upgrade to match latest version. If we only upgrade this repo, the pipeline still uses old versions of submodules which might not work.

**Comment by lquanx** [comment]
Is upgrading v2.3.0 required by the JIRA card ? If not, and if upgrading is necessary, how about we have a proper JIRA task regarding upgrade ? The task would suggest to upgrade all submodules, and pass reaggregation tests. 

Sometimes upgrading would break current system. For example, I remember it was fine to search resources with non-existing index before. But now it returns errors.

**Comment by nirojshrestha019** [comment]
> Is upgrading v2.3.0 required by the JIRA card ? If not, and if upgrading is necessary, how about we have a proper JIRA task regarding upgrade ? The task would suggest to upgrade all submodules, and pass reaggregation tests.
> 
> Sometimes upgrading would break current system. For example, I remember it was fine to search resources with non-existing index before. But now it returns errors.

Sounds good to me. I am reverting the upstream pull.

**Comment by Lin-002** [comment]
Hey Niroj, actually I meant we can have PR regarding CHP-151 only. We can manage upgrade later on. Is upgrading is required  in order to tackle CHP-151 ?

File: CHANGELOG.md (hunk 1 above)
Comment (nirojshrestha019) [reply CHANGELOG.md]: sounds good to me. I have reverted the upstream pull and have only included my cloudwatch alarm change in this PR.

**Comment by lquanx** [comment]
package-lock.json  shouldn't be committed.

**Comment by nirojshrestha019** [comment]
> package-lock.json shouldn't be committed.

My repo doesn't have package-lock.json file it's in Lee's repo.

**Comment by nirojshrestha019** [comment]
> > package-lock.json shouldn't be committed.
> 
> My repo doesn't have package-lock.json file it's in Lee's repo.

re added package-lock file

**Review by lwarren01 — APPROVED** [review]

**Pull request merged**